application-mocca-calendar-api/src/main/java/org/xwiki/contrib/moccacalendar/script/MoccaCalendarScriptService.java

```diff
@@ -284,6 +294,27 @@ public List<EventInstance> queryEvents(Date dateFrom, Date dateTo, String filter
         return events;
     }
 
+    /**
+     * Gets the union of events on a set of wikis.
```

Comment:
```suggestion
     * Gets the union of events on a set of wikis.
     *
```

Make sure you format the code using XWiki's code style.